source/cloud-native-platform/new-component/feature-flags.html.md.erb

```diff
@@ -30,7 +30,7 @@ End to end Process:
 ### Accessing LaunchDarkly
 
 Developers automatically get access by being part of 'DTS CFT Developers' or 'DTS SDS Developers', non-developers can get access with the 'DTS LaunchDarkly' group.
-see [onboarding](../../onboarding/person/index.html) process to request access. You will need to login from [myapps.microsoft.com](https://myapps.microsoft.com/hmcts.net).
+see [onboarding](https://hmcts.github.io/cloud-native-platform/onboarding/person/index.html) process to request access. You will need to login from [myapps.microsoft.com](https://myapps.microsoft.com/hmcts.net).
```

Comment:
links to this site should be relative or at most use a `/`
hmcts.github.io shouldn't be used as you may be testing locally and then end up on the live site